htsohm/simulation/xenon_loading.py

```diff
@@ -152,7 +152,6 @@ def run(run_id, material_id, helium_void_fraction):
                 if file_name_part in file:
                     output_file = os.path.join(output_subdir, file)
             print('OUTPUT FILE:\t%s' % output_file)
-#            output_file = os.path.join(output_dir, 'Output', 'System_0', filename)
```

Comment:
don't leave commented out lines in commit. This is why we `git diff --staged` before commits.